2018-03-13 14:58:49.523 ERROR 88192 --- [main] o.s.boot.SpringApplication               : Application startup failed

java.lang.IllegalStateException: Logback configuration error detected: 
ERROR in ch.qos.logback.core.joran.action.AppenderAction - Could not create an Appender of type [com.getsentry.raven.logback.SentryAppender]. ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type com.getsentry.raven.logback.SentryAppender
ERROR in ch.qos.logback.core.joran.spi.Interpreter@7:80 - ActionException in Action for tag [appender] ch.qos.logback.core.joran.spi.ActionException: ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type com.getsentry.raven.logback.SentryAppender
ERROR in ch.qos.logback.core.joran.spi.Interpreter@15:14 - no applicable action for [dsn], current ElementPath  is [[configuration][appender][dsn]]
ERROR in ch.qos.logback.core.joran.action.AppenderAction - Could not create an Appender of type [com.getsentry.raven.logback.SentryAppender]. ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type com.getsentry.raven.logback.SentryAppender
ERROR in ch.qos.logback.core.joran.spi.Interpreter@7:80 - ActionException in Action for tag [appender] ch.qos.logback.core.joran.spi.ActionException: ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type com.getsentry.raven.logback.SentryAppender
ERROR in ch.qos.logback.core.joran.spi.Interpreter@15:14 - no applicable action for [dsn], current ElementPath  is [[configuration][appender][dsn]]
	at org.springframework.boot.logging.logback.LogbackLoggingSystem.loadConfiguration(LogbackLoggingSystem.java:162)
	at org.springframework.boot.logging.AbstractLoggingSystem.initializeWithConventions(AbstractLoggingSystem.java:81)
	at org.springframework.boot.logging.AbstractLoggingSystem.initialize(AbstractLoggingSystem.java:59)
	at org.springframework.boot.logging.logback.LogbackLoggingSystem.initialize(LogbackLoggingSystem.java:115)
	at org.springframework.boot.logging.LoggingApplicationListener.initializeSystem(LoggingApplicationListener.java:303)
	at org.springframework.boot.logging.LoggingApplicationListener.initialize(LoggingApplicationListener.java:276)
	at org.springframework.boot.logging.LoggingApplicationListener.onApplicationEnvironmentPreparedEvent(LoggingApplicationListener.java:239)
	at org.springframework.boot.logging.LoggingApplicationListener.onApplicationEvent(LoggingApplicationListener.java:212)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:122)
	at org.springframework.boot.context.event.EventPublishingRunListener.environmentPrepared(EventPublishingRunListener.java:74)
	at org.springframework.boot.SpringApplicationRunListeners.environmentPrepared(SpringApplicationRunListeners.java:54)
	at org.springframework.boot.SpringApplication.prepareEnvironment(SpringApplication.java:325)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:296)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1118)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1107)
	at de.myDom.Main.main(Main.java:18)

2018-03-13 14:58:49.543  WARN 88192 --- [main] io.sentry.dsn.Dsn                        : *** Couldn't find a suitable DSN, Sentry operations will do nothing! See documentation: https://docs.sentry.io/clients/java/ ***
2018-03-13 14:58:49.553  WARN 88192 --- [main] io.sentry.DefaultSentryClientFactory     : No 'stacktrace.app.packages' was configured, this option is highly recommended as it affects stacktrace grouping and display on Sentry. See documentation: https://docs.sentry.io/clients/java/config/#in-application-stack-frames
2018-03-13 14:59:12.782 ERROR 89616 --- [main] o.s.boot.SpringApplication               : Application startup failed

java.lang.IllegalStateException: Logback configuration error detected: 
ERROR in ch.qos.logback.core.joran.spi.Interpreter@15:14 - no applicable action for [dsn], current ElementPath  is [[configuration][appender][dsn]]
ERROR in ch.qos.logback.core.joran.spi.Interpreter@15:14 - no applicable action for [dsn], current ElementPath  is [[configuration][appender][dsn]]
	at org.springframework.boot.logging.logback.LogbackLoggingSystem.loadConfiguration(LogbackLoggingSystem.java:162)
	at org.springframework.boot.logging.AbstractLoggingSystem.initializeWithConventions(AbstractLoggingSystem.java:81)
	at org.springframework.boot.logging.AbstractLoggingSystem.initialize(AbstractLoggingSystem.java:59)
	at org.springframework.boot.logging.logback.LogbackLoggingSystem.initialize(LogbackLoggingSystem.java:115)
	at org.springframework.boot.logging.LoggingApplicationListener.initializeSystem(LoggingApplicationListener.java:303)
	at org.springframework.boot.logging.LoggingApplicationListener.initialize(LoggingApplicationListener.java:276)
	at org.springframework.boot.logging.LoggingApplicationListener.onApplicationEnvironmentPreparedEvent(LoggingApplicationListener.java:239)
	at org.springframework.boot.logging.LoggingApplicationListener.onApplicationEvent(LoggingApplicationListener.java:212)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:122)
	at org.springframework.boot.context.event.EventPublishingRunListener.environmentPrepared(EventPublishingRunListener.java:74)
	at org.springframework.boot.SpringApplicationRunListeners.environmentPrepared(SpringApplicationRunListeners.java:54)
	at org.springframework.boot.SpringApplication.prepareEnvironment(SpringApplication.java:325)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:296)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1118)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1107)
	at de.myDom.Main.main(Main.java:18)

2018-03-13 14:59:12.797  WARN 89616 --- [main] io.sentry.dsn.Dsn                        : *** Couldn't find a suitable DSN, Sentry operations will do nothing! See documentation: https://docs.sentry.io/clients/java/ ***
2018-03-13 14:59:12.806  WARN 89616 --- [main] io.sentry.DefaultSentryClientFactory     : No 'stacktrace.app.packages' was configured, this option is highly recommended as it affects stacktrace grouping and display on Sentry. See documentation: https://docs.sentry.io/clients/java/config/#in-application-stack-frames
2018-03-13 15:05:05.973  INFO 79568 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 79568 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-13 15:05:05.998  WARN 79568 --- [main] io.sentry.dsn.Dsn                        : *** Couldn't find a suitable DSN, Sentry operations will do nothing! See documentation: https://docs.sentry.io/clients/java/ ***
2018-03-13 15:05:06.062  WARN 79568 --- [main] io.sentry.DefaultSentryClientFactory     : No 'stacktrace.app.packages' was configured, this option is highly recommended as it affects stacktrace grouping and display on Sentry. See documentation: https://docs.sentry.io/clients/java/config/#in-application-stack-frames
2018-03-13 15:05:06.072  INFO 79568 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-13 15:05:06.278  INFO 79568 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Tue Mar 13 15:05:06 CET 2018]; root of context hierarchy
2018-03-13 15:05:07.436  INFO 79568 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-13 15:05:07.449  INFO 79568 --- [main] de.myDom.Main                            : Started Main in 1.94 seconds (JVM running for 3.655)
2018-03-13 15:05:07.475  INFO 79568 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 16ms
2018-03-13 15:05:07.480  INFO 79568 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:05:07.484  INFO 79568 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:05:07.488  INFO 79568 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:05:07.492  INFO 79568 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:05:07.496  INFO 79568 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:05:07.500  INFO 79568 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:05:07.505  INFO 79568 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:05:07.509  INFO 79568 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:05:07.514  INFO 79568 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:05:07.516  INFO 79568 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 2ms
2018-03-13 15:05:07.516  INFO 79568 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-13 15:05:07.517  WARN 79568 --- [main] de.myDom.Service                         : 1. rausgeschrieben
2018-03-13 15:05:07.517  INFO 79568 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 1ms
2018-03-13 15:05:07.517  WARN 79568 --- [main] de.myDom.Service                         : 2. rausgeschrieben
2018-03-13 15:05:07.517  INFO 79568 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 0ms
2018-03-13 15:05:07.518  WARN 79568 --- [main] de.myDom.Service                         : 3. rausgeschrieben
2018-03-13 15:05:07.518  INFO 79568 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 0ms
2018-03-13 15:05:07.518  INFO 79568 --- [main] de.myDom.Main                            : Ende.
2018-03-13 15:05:07.520  INFO 79568 --- [Thread-8] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Tue Mar 13 15:05:06 CET 2018]; root of context hierarchy
2018-03-13 15:05:07.522  INFO 79568 --- [Thread-8] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-13 15:16:59.925  INFO 87988 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 87988 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-13 15:16:59.952  WARN 87988 --- [main] io.sentry.dsn.Dsn                        : *** Couldn't find a suitable DSN, Sentry operations will do nothing! See documentation: https://docs.sentry.io/clients/java/ ***
2018-03-13 15:16:59.962  WARN 87988 --- [main] io.sentry.DefaultSentryClientFactory     : No 'stacktrace.app.packages' was configured, this option is highly recommended as it affects stacktrace grouping and display on Sentry. See documentation: https://docs.sentry.io/clients/java/config/#in-application-stack-frames
2018-03-13 15:16:59.969  INFO 87988 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-13 15:17:00.071  INFO 87988 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Tue Mar 13 15:17:00 CET 2018]; root of context hierarchy
2018-03-13 15:17:01.332  INFO 87988 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-13 15:17:01.345  INFO 87988 --- [main] de.myDom.Main                            : Started Main in 1.837 seconds (JVM running for 3.457)
2018-03-13 15:17:01.372  INFO 87988 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 16ms
2018-03-13 15:17:01.377  INFO 87988 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:17:01.381  INFO 87988 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 3ms
2018-03-13 15:17:01.385  INFO 87988 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:17:01.389  INFO 87988 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:17:01.393  INFO 87988 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:17:01.397  INFO 87988 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 3ms
2018-03-13 15:17:01.402  INFO 87988 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:17:01.406  INFO 87988 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-13 15:17:01.410  INFO 87988 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 3ms
2018-03-13 15:17:01.412  INFO 87988 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 1ms
2018-03-13 15:17:01.412  INFO 87988 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-13 15:17:01.413  WARN 87988 --- [main] de.myDom.Service                         : 1. rausgeschrieben
2018-03-13 15:17:01.413  INFO 87988 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 0ms
2018-03-13 15:17:01.413  WARN 87988 --- [main] de.myDom.Service                         : 2. rausgeschrieben
2018-03-13 15:17:01.413  INFO 87988 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 0ms
2018-03-13 15:17:01.414  WARN 87988 --- [main] de.myDom.Service                         : 3. rausgeschrieben
2018-03-13 15:17:01.414  INFO 87988 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 0ms
2018-03-13 15:17:01.414  INFO 87988 --- [main] de.myDom.Main                            : Ende.
2018-03-13 15:17:01.415  INFO 87988 --- [Thread-7] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Tue Mar 13 15:17:00 CET 2018]; root of context hierarchy
2018-03-13 15:17:01.417  INFO 87988 --- [Thread-7] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-16 15:35:44.189 ERROR 112920 --- [main] o.s.boot.SpringApplication               : Application startup failed

java.lang.IllegalStateException: Logback configuration error detected: 
ERROR in ch.qos.logback.core.joran.action.AppenderAction - Could not create an Appender of type [io.sentryk.logback.SentryAppender]. ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type io.sentryk.logback.SentryAppender
ERROR in ch.qos.logback.core.joran.spi.Interpreter@7:71 - ActionException in Action for tag [appender] ch.qos.logback.core.joran.spi.ActionException: ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type io.sentryk.logback.SentryAppender
ERROR in ch.qos.logback.core.joran.action.AppenderRefAction - Could not find an appender named [SENTRY]. Did you define it below instead of above in the configuration file?
ERROR in ch.qos.logback.core.joran.action.AppenderRefAction - See http://logback.qos.ch/codes.html#appender_order for more details.
ERROR in ch.qos.logback.core.joran.action.AppenderAction - Could not create an Appender of type [io.sentryk.logback.SentryAppender]. ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type io.sentryk.logback.SentryAppender
ERROR in ch.qos.logback.core.joran.spi.Interpreter@7:71 - ActionException in Action for tag [appender] ch.qos.logback.core.joran.spi.ActionException: ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type io.sentryk.logback.SentryAppender
ERROR in ch.qos.logback.core.joran.action.AppenderRefAction - Could not find an appender named [SENTRY]. Did you define it below instead of above in the configuration file?
ERROR in ch.qos.logback.core.joran.action.AppenderRefAction - See http://logback.qos.ch/codes.html#appender_order for more details.
	at org.springframework.boot.logging.logback.LogbackLoggingSystem.loadConfiguration(LogbackLoggingSystem.java:162)
	at org.springframework.boot.logging.AbstractLoggingSystem.initializeWithConventions(AbstractLoggingSystem.java:81)
	at org.springframework.boot.logging.AbstractLoggingSystem.initialize(AbstractLoggingSystem.java:59)
	at org.springframework.boot.logging.logback.LogbackLoggingSystem.initialize(LogbackLoggingSystem.java:115)
	at org.springframework.boot.logging.LoggingApplicationListener.initializeSystem(LoggingApplicationListener.java:303)
	at org.springframework.boot.logging.LoggingApplicationListener.initialize(LoggingApplicationListener.java:276)
	at org.springframework.boot.logging.LoggingApplicationListener.onApplicationEnvironmentPreparedEvent(LoggingApplicationListener.java:239)
	at org.springframework.boot.logging.LoggingApplicationListener.onApplicationEvent(LoggingApplicationListener.java:212)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:122)
	at org.springframework.boot.context.event.EventPublishingRunListener.environmentPrepared(EventPublishingRunListener.java:74)
	at org.springframework.boot.SpringApplicationRunListeners.environmentPrepared(SpringApplicationRunListeners.java:54)
	at org.springframework.boot.SpringApplication.prepareEnvironment(SpringApplication.java:325)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:296)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1118)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1107)
	at de.myDom.Main.main(Main.java:18)

2018-03-16 15:40:33.064  INFO 113436 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 113436 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-16 15:40:33.120  INFO 113436 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-16 15:40:33.237  INFO 113436 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 15:40:33 CET 2018]; root of context hierarchy
2018-03-16 15:40:34.552  INFO 113436 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-16 15:40:34.566  INFO 113436 --- [main] de.myDom.Main                            : Started Main in 1.854 seconds (JVM running for 3.313)
2018-03-16 15:40:34.598  INFO 113436 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 21ms
2018-03-16 15:40:34.606  INFO 113436 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 7ms
2018-03-16 15:40:34.614  INFO 113436 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 8ms
2018-03-16 15:40:34.623  INFO 113436 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 8ms
2018-03-16 15:40:34.632  INFO 113436 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 9ms
2018-03-16 15:40:34.638  INFO 113436 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 15:40:34.643  INFO 113436 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:40:34.649  INFO 113436 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 15:40:34.654  INFO 113436 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 15:40:34.660  INFO 113436 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:40:34.662  INFO 113436 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 2ms
2018-03-16 15:40:34.663  INFO 113436 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-16 15:40:34.667  INFO 113436 --- [Thread-8] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 15:40:33 CET 2018]; root of context hierarchy
2018-03-16 15:40:34.669  INFO 113436 --- [Thread-8] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-16 15:40:35.668  WARN 113436 --- [Thread-2] io.sentry.connection.AsyncConnection     : Graceful shutdown took too much time, forcing the shutdown.
2018-03-16 15:40:35.669  WARN 113436 --- [Thread-2] io.sentry.connection.AsyncConnection     : 0 tasks failed to execute before shutdown.
2018-03-16 15:41:58.568  INFO 114052 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 114052 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-16 15:41:58.585  WARN 114052 --- [main] io.sentry.dsn.Dsn                        : *** Couldn't find a suitable DSN, Sentry operations will do nothing! See documentation: https://docs.sentry.io/clients/java/ ***
2018-03-16 15:41:58.592  WARN 114052 --- [main] io.sentry.DefaultSentryClientFactory     : No 'stacktrace.app.packages' was configured, this option is highly recommended as it affects stacktrace grouping and display on Sentry. See documentation: https://docs.sentry.io/clients/java/config/#in-application-stack-frames
2018-03-16 15:41:58.596  INFO 114052 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-16 15:41:58.669  INFO 114052 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 15:41:58 CET 2018]; root of context hierarchy
2018-03-16 15:41:59.870  INFO 114052 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-16 15:41:59.882  INFO 114052 --- [main] de.myDom.Main                            : Started Main in 1.709 seconds (JVM running for 3.213)
2018-03-16 15:41:59.907  INFO 114052 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 15ms
2018-03-16 15:41:59.911  INFO 114052 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 3ms
2018-03-16 15:41:59.915  INFO 114052 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 3ms
2018-03-16 15:41:59.920  INFO 114052 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:41:59.924  INFO 114052 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:41:59.927  INFO 114052 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 3ms
2018-03-16 15:41:59.931  INFO 114052 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 3ms
2018-03-16 15:41:59.935  INFO 114052 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 3ms
2018-03-16 15:41:59.939  INFO 114052 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 3ms
2018-03-16 15:41:59.944  INFO 114052 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:41:59.946  INFO 114052 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 2ms
2018-03-16 15:41:59.946  INFO 114052 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-16 15:41:59.947  WARN 114052 --- [main] de.myDom.Service                         : 1. rausgeschrieben
2018-03-16 15:41:59.947  INFO 114052 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 0ms
2018-03-16 15:41:59.952  INFO 114052 --- [Thread-8] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 15:41:58 CET 2018]; root of context hierarchy
2018-03-16 15:41:59.955  INFO 114052 --- [Thread-8] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-16 15:47:59.303  INFO 42136 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 42136 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-16 15:47:59.366  INFO 42136 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-16 15:47:59.487  INFO 42136 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 15:47:59 CET 2018]; root of context hierarchy
2018-03-16 15:48:00.739  INFO 42136 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-16 15:48:00.758  INFO 42136 --- [main] de.myDom.Main                            : Started Main in 2.065 seconds (JVM running for 3.738)
2018-03-16 15:48:00.781  INFO 42136 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 15ms
2018-03-16 15:48:00.786  INFO 42136 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:48:00.790  INFO 42136 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:48:00.796  INFO 42136 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 15:48:00.801  INFO 42136 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:48:00.806  INFO 42136 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:48:00.811  INFO 42136 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:48:00.816  INFO 42136 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 15:48:00.821  INFO 42136 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:48:00.825  INFO 42136 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:48:00.828  INFO 42136 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 2ms
2018-03-16 15:48:00.829  INFO 42136 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-16 15:48:00.830  WARN 42136 --- [main] de.myDom.Service                         : 1. rausgeschrieben
2018-03-16 15:48:00.830  INFO 42136 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 1ms
2018-03-16 15:48:00.830  WARN 42136 --- [main] de.myDom.Service                         : 2. rausgeschrieben
2018-03-16 15:48:00.830  INFO 42136 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 0ms
2018-03-16 15:48:00.831  WARN 42136 --- [main] de.myDom.Service                         : 3. rausgeschrieben
2018-03-16 15:48:00.831  INFO 42136 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 0ms
2018-03-16 15:48:00.831  INFO 42136 --- [main] de.myDom.Main                            : Ende.
2018-03-16 15:48:00.833  INFO 42136 --- [Thread-10] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 15:47:59 CET 2018]; root of context hierarchy
2018-03-16 15:48:00.835  INFO 42136 --- [Thread-10] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-16 15:48:01.834  WARN 42136 --- [Thread-4] io.sentry.connection.AsyncConnection     : Graceful shutdown took too much time, forcing the shutdown.
2018-03-16 15:48:01.835  WARN 42136 --- [Thread-4] io.sentry.connection.AsyncConnection     : 3 tasks failed to execute before shutdown.
2018-03-16 15:49:08.804  INFO 113632 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 113632 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-16 15:49:08.891  INFO 113632 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-16 15:49:09.007  INFO 113632 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 15:49:09 CET 2018]; root of context hierarchy
2018-03-16 15:49:10.747  INFO 113632 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-16 15:49:10.761  INFO 113632 --- [main] de.myDom.Main                            : Started Main in 2.344 seconds (JVM running for 4.111)
2018-03-16 15:49:10.786  INFO 113632 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 17ms
2018-03-16 15:49:10.791  INFO 113632 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:49:10.795  INFO 113632 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:49:10.801  INFO 113632 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 15:49:10.806  INFO 113632 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:49:10.816  INFO 113632 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 9ms
2018-03-16 15:49:10.820  INFO 113632 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:49:10.826  INFO 113632 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 15:49:10.832  INFO 113632 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:49:10.837  INFO 113632 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:49:10.840  INFO 113632 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 2ms
2018-03-16 15:49:10.841  INFO 113632 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-16 15:49:10.847  INFO 113632 --- [Thread-10] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 15:49:09 CET 2018]; root of context hierarchy
2018-03-16 15:49:10.849  INFO 113632 --- [Thread-10] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-16 15:49:11.847  WARN 113632 --- [Thread-3] io.sentry.connection.AsyncConnection     : Graceful shutdown took too much time, forcing the shutdown.
2018-03-16 15:49:11.848  WARN 113632 --- [Thread-3] io.sentry.connection.AsyncConnection     : 6 tasks failed to execute before shutdown.
2018-03-16 15:54:38.249  INFO 114784 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 114784 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-16 15:54:38.326  INFO 114784 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-16 15:54:38.494  INFO 114784 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 15:54:38 CET 2018]; root of context hierarchy
2018-03-16 15:54:39.762  INFO 114784 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-16 15:54:39.780  INFO 114784 --- [main] de.myDom.Main                            : Started Main in 1.889 seconds (JVM running for 3.354)
2018-03-16 15:54:39.818  INFO 114784 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 22ms
2018-03-16 15:54:39.824  INFO 114784 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 15:54:39.831  INFO 114784 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 15:54:39.839  INFO 114784 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 15:54:39.847  INFO 114784 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 15:54:39.853  INFO 114784 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 15:54:39.860  INFO 114784 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 15:54:39.866  INFO 114784 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 15:54:39.871  INFO 114784 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 15:54:39.876  INFO 114784 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:54:39.879  INFO 114784 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 2ms
2018-03-16 15:54:39.880  INFO 114784 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-16 15:54:39.881  WARN 114784 --- [main] de.myDom.Service                         : 1. rausgeschrieben
2018-03-16 15:54:39.882  INFO 114784 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 1ms
2018-03-16 15:54:39.887  INFO 114784 --- [Thread-8] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 15:54:38 CET 2018]; root of context hierarchy
2018-03-16 15:54:39.888  INFO 114784 --- [Thread-8] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-16 15:54:40.899  WARN 114784 --- [Thread-2] io.sentry.connection.AsyncConnection     : Graceful shutdown took too much time, forcing the shutdown.
2018-03-16 15:54:40.900  WARN 114784 --- [Thread-2] io.sentry.connection.AsyncConnection     : 0 tasks failed to execute before shutdown.
2018-03-16 15:58:49.047  INFO 116220 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 116220 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-16 15:58:49.102  INFO 116220 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-16 15:58:49.209  INFO 116220 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 15:58:49 CET 2018]; root of context hierarchy
2018-03-16 15:58:50.530  INFO 116220 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-16 15:58:50.543  INFO 116220 --- [main] de.myDom.Main                            : Started Main in 1.823 seconds (JVM running for 3.308)
2018-03-16 15:58:50.567  INFO 116220 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 15ms
2018-03-16 15:58:50.572  INFO 116220 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:58:50.579  INFO 116220 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 15:58:50.588  INFO 116220 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 7ms
2018-03-16 15:58:50.610  INFO 116220 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 22ms
2018-03-16 15:58:50.616  INFO 116220 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 15:58:50.620  INFO 116220 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:58:50.624  INFO 116220 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 3ms
2018-03-16 15:58:50.629  INFO 116220 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:58:50.633  INFO 116220 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:58:50.636  INFO 116220 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 2ms
2018-03-16 15:58:50.637  INFO 116220 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-16 15:58:50.642  INFO 116220 --- [Thread-9] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 15:58:49 CET 2018]; root of context hierarchy
2018-03-16 15:58:50.644  INFO 116220 --- [Thread-9] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-16 15:58:51.642  WARN 116220 --- [Thread-3] io.sentry.connection.AsyncConnection     : Graceful shutdown took too much time, forcing the shutdown.
2018-03-16 15:58:51.643  WARN 116220 --- [Thread-3] io.sentry.connection.AsyncConnection     : 4 tasks failed to execute before shutdown.
2018-03-16 15:59:44.839  INFO 106780 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 106780 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-16 15:59:44.933  INFO 106780 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-16 15:59:45.068  INFO 106780 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 15:59:45 CET 2018]; root of context hierarchy
2018-03-16 15:59:46.401  INFO 106780 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-16 15:59:46.414  INFO 106780 --- [main] de.myDom.Main                            : Started Main in 1.907 seconds (JVM running for 3.392)
2018-03-16 15:59:46.447  INFO 106780 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 21ms
2018-03-16 15:59:46.455  INFO 106780 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 8ms
2018-03-16 15:59:46.461  INFO 106780 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 15:59:46.468  INFO 106780 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 15:59:46.475  INFO 106780 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 15:59:46.481  INFO 106780 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 15:59:46.485  INFO 106780 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 15:59:46.491  INFO 106780 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 15:59:46.497  INFO 106780 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 15:59:46.503  INFO 106780 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 15:59:46.506  INFO 106780 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 2ms
2018-03-16 15:59:46.507  INFO 106780 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-16 15:59:46.508  WARN 106780 --- [main] de.myDom.Service                         : 1. rausgeschrieben
2018-03-16 15:59:46.508  INFO 106780 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 0ms
2018-03-16 15:59:46.509  WARN 106780 --- [main] de.myDom.Service                         : 2. rausgeschrieben
2018-03-16 15:59:46.509  INFO 106780 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 0ms
2018-03-16 15:59:46.509  WARN 106780 --- [main] de.myDom.Service                         : 3. rausgeschrieben
2018-03-16 15:59:46.510  INFO 106780 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 1ms
2018-03-16 15:59:46.510  INFO 106780 --- [main] de.myDom.Main                            : Ende.
2018-03-16 15:59:46.512  INFO 106780 --- [Thread-8] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 15:59:45 CET 2018]; root of context hierarchy
2018-03-16 15:59:46.514  INFO 106780 --- [Thread-8] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-16 15:59:47.519  WARN 106780 --- [Thread-2] io.sentry.connection.AsyncConnection     : Graceful shutdown took too much time, forcing the shutdown.
2018-03-16 15:59:47.523  WARN 106780 --- [Thread-2] io.sentry.connection.AsyncConnection     : 7 tasks failed to execute before shutdown.
2018-03-16 16:51:26.445  INFO 116076 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 116076 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-16 16:51:26.498  INFO 116076 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-16 16:51:26.648  INFO 116076 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 16:51:26 CET 2018]; root of context hierarchy
2018-03-16 16:51:28.375  INFO 116076 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-16 16:51:28.407  INFO 116076 --- [main] de.myDom.Main                            : Started Main in 2.294 seconds (JVM running for 3.703)
2018-03-16 16:51:28.474  INFO 116076 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 51ms
2018-03-16 16:51:28.481  INFO 116076 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 16:51:28.492  INFO 116076 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 9ms
2018-03-16 16:51:28.505  INFO 116076 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 11ms
2018-03-16 16:51:28.515  INFO 116076 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 10ms
2018-03-16 16:51:28.527  INFO 116076 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 10ms
2018-03-16 16:51:28.536  INFO 116076 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 9ms
2018-03-16 16:51:28.550  INFO 116076 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 11ms
2018-03-16 16:51:28.560  INFO 116076 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 9ms
2018-03-16 16:51:28.568  INFO 116076 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 8ms
2018-03-16 16:51:28.571  INFO 116076 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 2ms
2018-03-16 16:51:28.573  INFO 116076 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-16 16:51:28.574  WARN 116076 --- [main] de.myDom.Service                         : 1. rausgeschrieben
2018-03-16 16:51:28.575  INFO 116076 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 1ms
2018-03-16 16:51:28.575  WARN 116076 --- [main] de.myDom.Service                         : 2. rausgeschrieben
2018-03-16 16:51:28.575  INFO 116076 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 0ms
2018-03-16 16:51:28.584  INFO 116076 --- [Thread-10] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 16:51:26 CET 2018]; root of context hierarchy
2018-03-16 16:51:28.587  INFO 116076 --- [Thread-10] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-16 16:51:29.585  WARN 116076 --- [Thread-3] io.sentry.connection.AsyncConnection     : Graceful shutdown took too much time, forcing the shutdown.
2018-03-16 16:51:29.586  WARN 116076 --- [Thread-3] io.sentry.connection.AsyncConnection     : 6 tasks failed to execute before shutdown.
2018-03-16 16:54:01.381  INFO 119712 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 119712 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-16 16:54:01.453  INFO 119712 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-16 16:54:01.660  INFO 119712 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 16:54:01 CET 2018]; root of context hierarchy
2018-03-16 16:54:03.213  INFO 119712 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-16 16:54:03.232  INFO 119712 --- [main] de.myDom.Main                            : Started Main in 2.207 seconds (JVM running for 4.195)
2018-03-16 16:54:03.271  INFO 119712 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 26ms
2018-03-16 16:54:03.276  INFO 119712 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 16:54:03.283  INFO 119712 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 16:54:03.289  INFO 119712 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 16:54:03.295  INFO 119712 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 16:54:03.300  INFO 119712 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 16:54:03.306  INFO 119712 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 16:54:03.312  INFO 119712 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 16:54:03.318  INFO 119712 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 16:54:03.325  INFO 119712 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-16 16:54:03.328  INFO 119712 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 2ms
2018-03-16 16:54:03.329  INFO 119712 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-16 16:54:03.330  WARN 119712 --- [main] de.myDom.Service                         : 1. rausgeschrieben
2018-03-16 16:54:03.331  INFO 119712 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 1ms
2018-03-16 16:54:03.337  INFO 119712 --- [Thread-10] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 16:54:01 CET 2018]; root of context hierarchy
2018-03-16 16:54:03.340  INFO 119712 --- [Thread-10] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-16 16:54:04.341  WARN 119712 --- [Thread-3] io.sentry.connection.AsyncConnection     : Graceful shutdown took too much time, forcing the shutdown.
2018-03-16 16:54:04.342  WARN 119712 --- [Thread-3] io.sentry.connection.AsyncConnection     : 16 tasks failed to execute before shutdown.
2018-03-16 16:58:20.626  INFO 100172 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 100172 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-16 16:58:20.756  INFO 100172 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-16 16:58:20.881  INFO 100172 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 16:58:20 CET 2018]; root of context hierarchy
2018-03-16 16:58:22.589  INFO 100172 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-16 16:58:22.605  INFO 100172 --- [main] de.myDom.Main                            : Started Main in 2.308 seconds (JVM running for 3.78)
2018-03-16 16:58:22.630  INFO 100172 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 16ms
2018-03-16 16:58:22.635  INFO 100172 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 3ms
2018-03-16 16:58:22.640  INFO 100172 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 16:58:22.645  INFO 100172 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 16:58:22.655  INFO 100172 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 10ms
2018-03-16 16:58:22.661  INFO 100172 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 16:58:22.695  INFO 100172 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 32ms
2018-03-16 16:58:22.720  INFO 100172 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 23ms
2018-03-16 16:58:22.724  INFO 100172 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-16 16:58:22.730  INFO 100172 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-16 16:58:22.732  INFO 100172 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 2ms
2018-03-16 16:58:22.733  INFO 100172 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-16 16:58:22.739  INFO 100172 --- [Thread-10] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Fri Mar 16 16:58:20 CET 2018]; root of context hierarchy
2018-03-16 16:58:22.741  INFO 100172 --- [Thread-10] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-16 16:58:23.226  WARN 100172 --- [sentry-pool-1-thread-3] i.s.c.AbstractConnection.lockdown        : Initiated a temporary lockdown because of exception: An exception occurred while submitting the event to the Sentry server.
2018-03-16 16:58:23.235 ERROR 100172 --- [sentry-pool-1-thread-3] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-16 16:58:23.250 ERROR 100172 --- [sentry-pool-1-thread-1] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-16 16:58:23.254 ERROR 100172 --- [sentry-pool-1-thread-2] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-16 16:58:23.477 ERROR 100172 --- [sentry-pool-1-thread-4] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-19 16:59:48.294  INFO 119580 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 119580 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-19 16:59:48.356  INFO 119580 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-19 16:59:48.464  INFO 119580 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@358ee631: startup date [Mon Mar 19 16:59:48 CET 2018]; root of context hierarchy
2018-03-19 16:59:50.181  INFO 119580 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-19 16:59:50.201  INFO 119580 --- [main] de.myDom.Main                            : Started Main in 2.469 seconds (JVM running for 5.268)
2018-03-19 16:59:50.272  INFO 119580 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 59ms
2018-03-19 16:59:50.285  INFO 119580 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 13ms
2018-03-19 16:59:50.301  INFO 119580 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 16ms
2018-03-19 16:59:50.312  INFO 119580 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 9ms
2018-03-19 16:59:50.321  INFO 119580 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 8ms
2018-03-19 16:59:50.329  INFO 119580 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 8ms
2018-03-19 16:59:50.336  INFO 119580 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-19 16:59:50.352  INFO 119580 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 14ms
2018-03-19 16:59:50.367  INFO 119580 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 14ms
2018-03-19 16:59:50.392  INFO 119580 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 24ms
2018-03-19 16:59:50.397  INFO 119580 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 4ms
2018-03-19 16:59:50.399  INFO 119580 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-19 16:59:50.400  WARN 119580 --- [main] de.myDom.Service                         : 1. rausgeschrieben
2018-03-19 16:59:50.400  INFO 119580 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 1ms
2018-03-19 16:59:50.412  INFO 119580 --- [Thread-12] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@358ee631: startup date [Mon Mar 19 16:59:48 CET 2018]; root of context hierarchy
2018-03-19 16:59:50.415  INFO 119580 --- [Thread-12] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-19 16:59:50.726  WARN 119580 --- [sentry-pool-1-thread-2] i.s.c.AbstractConnection.lockdown        : Initiated a temporary lockdown because of exception: An exception occurred while submitting the event to the Sentry server.
2018-03-19 16:59:50.730 ERROR 119580 --- [sentry-pool-1-thread-2] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-19 16:59:50.731 ERROR 119580 --- [sentry-pool-1-thread-1] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-19 16:59:50.745 ERROR 119580 --- [sentry-pool-1-thread-3] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-19 16:59:50.800 ERROR 119580 --- [sentry-pool-1-thread-4] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-19 17:27:33.198  INFO 125748 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 125748 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-19 17:27:33.259  INFO 125748 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-19 17:27:33.363  INFO 125748 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Mon Mar 19 17:27:33 CET 2018]; root of context hierarchy
2018-03-19 17:27:35.066  INFO 125748 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-19 17:27:35.080  INFO 125748 --- [main] de.myDom.Main                            : Started Main in 2.245 seconds (JVM running for 3.83)
2018-03-19 17:27:35.108  INFO 125748 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 18ms
2018-03-19 17:27:35.113  INFO 125748 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-19 17:27:35.118  INFO 125748 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-19 17:27:35.126  INFO 125748 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-19 17:27:35.133  INFO 125748 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 6ms
2018-03-19 17:27:35.138  INFO 125748 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-19 17:27:35.146  INFO 125748 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 7ms
2018-03-19 17:27:35.151  INFO 125748 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-19 17:27:35.156  INFO 125748 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-19 17:27:35.161  INFO 125748 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-19 17:27:35.163  INFO 125748 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 1ms
2018-03-19 17:27:35.164  INFO 125748 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-19 17:27:35.165  WARN 125748 --- [main] de.myDom.Service                         : 1. rausgeschrieben
2018-03-19 17:27:35.166  INFO 125748 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 1ms
2018-03-19 17:27:35.170  INFO 125748 --- [Thread-10] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Mon Mar 19 17:27:33 CET 2018]; root of context hierarchy
2018-03-19 17:27:35.173  INFO 125748 --- [Thread-10] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-19 17:27:35.583  WARN 125748 --- [sentry-pool-1-thread-1] i.s.c.AbstractConnection.lockdown        : Initiated a temporary lockdown because of exception: An exception occurred while submitting the event to the Sentry server.
2018-03-19 17:27:35.589 ERROR 125748 --- [sentry-pool-1-thread-1] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-19 17:27:35.599 ERROR 125748 --- [sentry-pool-1-thread-3] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-19 17:27:35.603 ERROR 125748 --- [sentry-pool-1-thread-2] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-19 17:27:35.758 ERROR 125748 --- [sentry-pool-1-thread-4] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-19 17:31:37.075  INFO 127696 --- [main] de.myDom.Main                            : Starting Main on ogitnb187 with PID 127696 (C:\Users\klinkmann\Desktop\annotProj\target\classes started by klinkmann in C:\Users\klinkmann\Desktop\annotProj)
2018-03-19 17:31:37.167  INFO 127696 --- [main] de.myDom.Main                            : The following profiles are active: dev
2018-03-19 17:31:37.397  INFO 127696 --- [main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Mon Mar 19 17:31:37 CET 2018]; root of context hierarchy
2018-03-19 17:31:38.725  INFO 127696 --- [main] o.s.j.e.a.AnnotationMBeanExporter        : Registering beans for JMX exposure on startup
2018-03-19 17:31:38.741  INFO 127696 --- [main] de.myDom.Main                            : Started Main in 2.078 seconds (JVM running for 3.898)
2018-03-19 17:31:38.768  INFO 127696 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 16ms
2018-03-19 17:31:38.773  INFO 127696 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-19 17:31:38.777  INFO 127696 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-19 17:31:38.782  INFO 127696 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-19 17:31:38.786  INFO 127696 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 3ms
2018-03-19 17:31:38.791  INFO 127696 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 5ms
2018-03-19 17:31:38.795  INFO 127696 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 3ms
2018-03-19 17:31:38.799  INFO 127696 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-19 17:31:38.803  INFO 127696 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-19 17:31:38.807  INFO 127696 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.writePerson(Person) executed in 4ms
2018-03-19 17:31:38.809  INFO 127696 --- [main] de.myDom.aspect.ExampleAspect            : Person de.myDom.Service.readPerson() executed in 1ms
2018-03-19 17:31:38.810  INFO 127696 --- [main] de.myDom.Main                            : Person{name='Arno', auto=Auto{marke='BMW', model='325', ps=218}}
2018-03-19 17:31:38.811  WARN 127696 --- [main] de.myDom.Service                         : 1. rausgeschrieben
2018-03-19 17:31:38.811  INFO 127696 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 0ms
2018-03-19 17:31:38.811  WARN 127696 --- [main] de.myDom.Service                         : 2. rausgeschrieben
2018-03-19 17:31:38.812  INFO 127696 --- [main] de.myDom.aspect.ExampleAspect            : void de.myDom.Service.nothing() executed in 1ms
2018-03-19 17:31:38.815  INFO 127696 --- [Thread-10] s.c.a.AnnotationConfigApplicationContext : Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@ec756bd: startup date [Mon Mar 19 17:31:37 CET 2018]; root of context hierarchy
2018-03-19 17:31:38.817  INFO 127696 --- [Thread-10] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2018-03-19 17:31:38.821  WARN 127696 --- [sentry-pool-1-thread-3] i.s.c.AbstractConnection.lockdown        : Initiated a temporary lockdown because of exception: An exception occurred while submitting the event to the Sentry server.
2018-03-19 17:31:38.823 ERROR 127696 --- [sentry-pool-1-thread-3] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-19 17:31:38.824 ERROR 127696 --- [sentry-pool-1-thread-2] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-19 17:31:38.919 ERROR 127696 --- [sentry-pool-1-thread-1] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

2018-03-19 17:31:39.285 ERROR 127696 --- [sentry-pool-1-thread-4] io.sentry.connection.AsyncConnection     : An exception occurred while sending the event to Sentry.

io.sentry.connection.ConnectionException: An exception occurred while submitting the event to the Sentry server.
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:205)
	at io.sentry.connection.AbstractConnection.send(AbstractConnection.java:75)
	at io.sentry.connection.AsyncConnection$EventSubmitter.run(AsyncConnection.java:185)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
	at sun.security.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1959)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:302)
	at sun.security.ssl.Handshaker.fatalSE(Handshaker.java:296)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1514)
	at sun.security.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:216)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
	at sun.security.ssl.Handshaker.process_record(Handshaker.java:961)
	at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:1072)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1385)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1413)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1397)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:559)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:185)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at io.sentry.connection.HttpConnection.doSend(HttpConnection.java:157)
	... 5 common frames omitted
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
	at sun.security.validator.Validator.validate(Validator.java:260)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:324)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:229)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:124)
	at sun.security.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1496)
	... 16 common frames omitted
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:392)
	... 22 common frames omitted

